CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [6.3.1] - 2020-07-06
```

Comment:
I would prefer 6.4.0 - there are some reasonably fickle changes in here which have been proven to be error prove :(

Reply:
They were significant changes, but still felt like a patch release to me overall. I'll bump it to a minor release though.